salt/modules/artifactory.py

```diff
@@ -38,7 +38,7 @@ def __virtual__():
         return True
 
 
-def get_latest_snapshot(artifactory_url, repository, group_id, artifact_id, packaging, target_dir='/tmp', target_file=None, classifier=None, username=None, password=None):
+def get_latest_snapshot(artifactory_url, repository, group_id, artifact_id, packaging, target_dir='/tmp', target_file=None, classifier=None, username=None, password=None, use_literal_group_id=False):
```

Comment:
Yes, we are currently trying to use this for a Windows 2008 R2 minion and I've also been testing on a Windows 7 machine. It seems to work fine as is, though I don't think our use case actually uses that function.